cms/templates/error.html

```diff
@@ -28,17 +28,17 @@
   <article class="error-prompt">
     % if error == '404':
       <h1>${_("The Page You Requested Page Cannot be Found")}</h1>
-<<<<<<< HEAD
-      <p class="description">${_("We're sorry. We couldn't find the Studio page you're looking for. You may want to return to the Studio Dashboard and try again. If you are still having problems accessing things, please feel free to {link_start}contact Studio support{link_end} for further help.").format(
-          link_start='<a href="https://groups.google.com/d/forum/openedx-studio" target="_blank" title="{title}">'.format(title=_("Use our feedback tool, Tender, to share your feedback")),
-          link_end='</a>',
-        )}</p>
-    % elif error == '500':
-      <h1>${_("The Server Encountered an Error")}</h1>
-      <p class="description">${_("We're sorry. There was a problem with the server while trying to process your last request. You may want to return to the Studio Dashboard or try this request again. If you are still having problems accessing things, please feel free to {link_start}contact Studio support{link_end} for further help.").format(
-          link_start='<a href="https://groups.google.com/d/forum/openedx-studio" target="_blank" title="{title}">'.format(title=_("Use our feedback tool, Tender, to share your feedback")),
-          link_end='</a>',
-=======
+<!-- <<<<<<< HEAD -->
```

Comment:
Are you sure?

The difference I'm seeing is that they've parameterized the term "Studio".  While it still defaults to "Studio", this allows you to change it so it displays something like "Please contact Stanford Online Course Creator support..." instead of "Please contact Studio support..."

It also looks like the contact link is now customizable too.

This change is propagated throughout the codebase, so I think we should avoid forking here if we don't need to; I think the text is still the same